### ISSUE TITLE
Device doesn't show up when loaded/imported if no device was previously loaded (Trac #138)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#142]
Fixed in 6b814143e2e19239e2692fec3fe6625ce73799da

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/138
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "Fixed in 6b814143e2e19239e2692fec3fe6625ce73799da", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Device doesn't show up when loaded/imported if no device was previously loaded", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2013-04-09T17:08:46", 
    "milestone": "Microdrop 1.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20130409T17:15:48: 
* ryan changed description from "" to "Fixed in 6b814143e2e19239e2692fec3fe6625ce73799da"
* ryan changed resolution from "" to "fixed"
* ryan changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "" to "microdrop"
